stdlib/universe/pivot_test.go

```diff
@@ -1163,7 +1309,7 @@ func TestPivot2_Process(t *testing.T) {
 						{Label: "grouper", Type: flux.TString},
 					},
 					Data: [][]interface{}{
-						{execute.Time(1), 2.0, "m1", "f2", "B"},
+						{execute.Time(2), 4.0, "m1", "f2", "A"},
```

Comment:
These tables now have to be sorted by groupKey for tests to pass.

Reply:
When I see someone comment on their own PR to explain something, that's usually a sign to me that those might be good comments _in_ the code. I think this holds up here very well.

Reply:
fair enough. I can do that.